aws/application/parameters/production-maat-cd-api-pipeline.json

```diff
@@ -18,6 +18,7 @@
     "pLaaProsecutionConcludedQueue": "crime-apps-production-prosecution-concluded-queue",
     "pCdaOauthUrl": "https://laa-court-data-adaptor.apps.live-1.cloud-platform.service.justice.gov.uk/oauth/token",
     "pCdaLaaStatusUrl": "https://laa-court-data-adaptor.apps.live-1.cloud-platform.service.justice.gov.uk/api/internal/v1/representation_orders",
+    "pCdaHearingUrl": "https://laa-court-data-adaptor.apps.live-1.cloud-platform.service.justice.gov.uk/api/internal/v1/hearings/{hearingId}",
```

Comment:
Can we call CDA's live cluster URL instead of live-1 as it is being depricated? I guess this will apply to LAA Status URL too.  Check with @laurentqro  for more details.

Reply:
I've tagged @laurentqro in the slack to sanity check all the CDA urls :)

Reply:
All CDA urls updated.